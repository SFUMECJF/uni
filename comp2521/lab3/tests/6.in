0
-15
-13
-14
-11
-9
-10
-12
-7
-5
-6
-3
-1
-2
-4
-8

