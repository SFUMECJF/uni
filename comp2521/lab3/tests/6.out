Tree:
    0
   /
 -15
   \
   -13
   / \
  /   \
-14   -11
      / \
     /   \
   -12   -9
         / \
        /   \
      -10   -7
            / \
           /   \
          -8   -5
               / \
              /   \
             -6   -3
                  / \
                 /   \
                -4   -1
                     /
                    -2

#nodes = 16,  #leaves = 7
Infix  : -15 -14 -13 -12 -11 -10 -9 -8 -7 -6 -5 -4 -3 -2 -1 0 
Prefix : 0 -15 -13 -14 -11 -12 -9 -10 -7 -8 -5 -6 -3 -4 -1 -2 
Postfix: -14 -12 -10 -8 -6 -4 -2 -1 -3 -5 -7 -9 -11 -13 -15 0 
ByLevel: 0 -15 -13 -14 -11 -12 -9 -10 -7 -8 -5 -6 -3 -4 -1 -2 
